### PR TITLE
Use postbuild step, update version to v0.1.0-alpha.3

### PR DESCRIPTION
**Pull request by akshay-ap**
Changes in PR:
- Use postbuild step
- update version to v0.1.0-alpha.3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
     "lint": "yarn lint:sol && yarn lint:ts",
     "lint:sol": "solhint 'contracts/**/*.sol'",
     "lint:ts": "eslint 'test/**/*.ts' --max-warnings 0 --fix",
-    "typechain": "TS_NODE_TRANSPILE_ONLY=true yarn hardhat typechain",
```
Comment (rmeissner) [inline package.json:31]: why was this changed?

**Review by rmeissner — APPROVED** [review]
Tested this and it works

File: package.json (hunk 1 above)
Comment (akshay-ap) [reply package.json:31]: restored change to previous content i.e. `TS_NODE_TRANSPILE_ONLY=true hardhat typechain`. `yarn` was added as a fix in alpha.2 but it didn't work
